query-node/mappings/src/council.ts

```diff
@@ -48,6 +48,10 @@ import {
   ElectedCouncil,
   CastVote,
   CandidacyNoteMetadata,
+  CandidacyStatusActive,
+  CandidacyStatusWithdrawn,
+  CandidacyStatusElected,
+  CandidacyStatusLost,
```

Comment:
It seems like `CandidacyStatusLost` got **lost** during some conflict resolutions(?).
If I compere this PR branch to `thesan/olympia` it looks like there is some code missing in `council_NewCouncilElected` handler.